doc/ref_model/chapters/glossary.md

```diff
@@ -37,7 +37,6 @@ To help guide the reader, this glossary provides an introduction to the terminol
   - **Storage Workloads**: are all tasks related to disk storage (either SSD or HDD or other).  Examples range from non-intensive router logging to more intensive database read/write operations.
 - **Virtual Machine (VM)**: virtualised computation environment that behaves like a physical computer/server.
   >_*Note:*_ A **VM** consists of all of the components (processor (CPU), memory, storage, interfaces/ports, etc.) of a physical computer/server. It is created using Instance Type together with sizing information or Compute Flavour.
-- **Instance type**: Specifies a set of virtualized hardware resources and capabilities used for the creation of a virtual compute on which a workload runs on, includes capability specifications characterizing compute, storage and memory.
```

Comment:
If the intent is to replace *instance type* definition with *NFVI profile*, please open issues to replace this term at other places (RA/RI/RC).

Reply:
already in here: https://github.com/cntt-n/CNTT/issues/1209